app/serializers/v1/workflow/process_serializer.rb

```diff
@@ -0,0 +1,16 @@
+class ProcessSerializer
+  include JSONAPI::Serializer
+
+  attributes :title, :effort, :categories, :status, :position #, :assignee
+
+  attribute :workflow do |object|
+    { name: object.workflow.name, description: object.workflow.description }
+  end
+
+  attribute :steps do |object|
+    object.steps.map do |step|
+      { title: step.title, completed: step.completed, kind: step.kind, resource_url: step.resource_url, position: step.position }
+    end
+  end
+end
+
```

Comment:
So the way I've been doing relationships is with the `has_many`
For example:
https://github.com/keithtom/wildflower-platform/blob/main/app/serializers/v1/school_serializer.rb
BTW, there's a bug so you have to write it like that even though it should be able to figure things out.  You can read more about it in this readme: https://github.com/keithtom/wildflower-platform/blob/main/app/serializers/README.md

It changes the JSON structure and is a bit more annoying for Taylor but I think this makes everything "standard JSON compliant API".   The con is that it isn't as convenient, the pro is that we have a standard way to interact w/ our api and we don't have to really think about it.  I kinda prefer just going w/ this method.

Reply:
ok, i was worried about consistency as well